packages/sdk/src/types/ClientSide.ts

```diff
@@ -121,6 +142,23 @@ export interface VoucherifyRedeemInputs {
 	voucherifyAmount: string
 	voucherifyTracking: string
 }
+
+export interface VoucherifyValidateInputsState {
```

Comment:
Let's please move those types which are strictly related to widget to some new file in `packages/react-widget` dir. It does not make sense for someone who only downaloaded SDK package to see those types.